internal/io/streams.go

```diff
@@ -69,7 +70,7 @@ func (s *Streams) Wrap(fn func() error) error {
 	// Log any panics to ttyout, since otherwise they will be lost to os.Stderr.
 	defer func() {
 		if r := recover(); r != nil {
-			fmt.Fprintln(s.TTYOut, r)
+			fmt.Fprintln(s.TTYOut, r, string(debug.Stack()))
```

Comment:
Intended to leave this in?

Reply:
yes. the recover alone isn't super useful.